### PR TITLE
IWPT 1991 to 2000

### PR DESCRIPTION
**Pull request by danielgildea**
This branch includes xml files generated from Kilian's ingest_tsv.py script.
PDFs for papers are at: https://www.cs.rochester.edu/u/gildea/iwpt.tgz
These PDFs were extracted from the OCR_Optimized directory referenced in 
Issue #355.

### CODE REVIEW

File: data/xml/1991.iwpt.xml [hunk 1]
```diff
@@ -0,0 +1,241 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="1991.iwpt">
+  <volume id="1">
+    <meta>
+      <booktitle>Second International Workshop on Parsing Technologies</booktitle>
+      <publisher>ACL</publisher>
```
Comment (mjpost) [inline data/xml/1991.iwpt.xml:6]: This should maybe be "Association for Computational Linguistics" (here and throughout)?

File: data/xml/1995.iwpt.xml [hunk 2]
```diff
@@ -0,0 +1,287 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="1995.iwpt">
+  <volume id="1">
+    <meta>
+      <booktitle>Fourth International Workshop on Parsing Technologies</booktitle>
+      <publisher>ACL</publisher>
+      <address>Prague and Karlovy Vary, Czech Republic</address>
+      <month>September 20-24</month>
```
Comment (mjpost) [inline data/xml/1995.iwpt.xml:8]: Looks like an editor got mis-parsed.

File: data/xml/1993.iwpt.xml [hunk 3]
```diff
@@ -0,0 +1,236 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="1993.iwpt">
+  <volume id="1">
+    <meta>
+      <booktitle>Third International Workshop on Parsing Technologies</booktitle>
+      <publisher>ACL</publisher>
+      <address>Tilburg, Netherlands and Durbuy, Belgium</address>
+      <month>August 10-13</month>
```
Comment (mjpost) [inline data/xml/1993.iwpt.xml:8]: Why is Durbuy here? [The official page](https://www.sigparse.org/meetings.html) lists Tilburg only.

File: data/xml/1991.iwpt.xml (hunk 1 above)
Comment (danielgildea) [reply data/xml/1991.iwpt.xml]: fixed

File: data/xml/1995.iwpt.xml (hunk 2 above)
Comment (danielgildea) [reply data/xml/1995.iwpt.xml:8]: no, it's a town ("Charles's Baths").  they had a tradition of having the second part of the conference in a resort/tourist area.

File: data/xml/1993.iwpt.xml (hunk 3 above)
Comment (danielgildea) [reply data/xml/1993.iwpt.xml:8]: second part of conference was in Durbuy

File: data/xml/1991.iwpt.xml [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+    </paper>
+    <paper id="5">
+      <title>Preprocessing and lexicon design for parsing technical text</title>
+      <author><first>Robert P.</first><last>Futrelle</last></author>
+      <author><first>Christopher E.</first><last>Dunn</last></author>
+      <author><first>Debra S.</first><last>Ellis</last></author>
+      <author><first>Jr.</first><last>Maurice J. Pescitelli</last></author>
+      <pages>31-40</pages>
```
Comment (mjpost) [inline data/xml/1991.iwpt.xml:61]: Name parsing.

File: data/xml/1991.iwpt.xml [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+    </paper>
+    <paper id="3">
+      <title>PARSING = PARSIMONIOUS COVERING?</title>
+      <author><first>Venu</first><last>Dasigi</last></author>
```
Comment (mjpost) [inline data/xml/1991.iwpt.xml:43]: Editorial decision whether to normal-case this

File: data/xml/1991.iwpt.xml [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+    </paper>
+    <paper id="13">
+      <title>Phonological Analysis and Opaque Rule Orders</title>
+      <author><first>Michael</first><last>Maxwell</last></author>
+      <pages>110-116</pages>
+      <url>1991.iwpt-1.13</url>
+      <abstract>General morphological/ phonological analysis using ordered phonological rules has appeared to be computationally expensive, because ambiguities in feature values arising when phonological rules are “un-applied” multiply with additional rules. But in fact those ambiguities can be largely ignored until lexical lookup, since the underlying values of altered features are needed only in the case of rare opaque rule orderings, and not always then.</abstract>
+    </paper>
```
Comment (mjpost) [inline data/xml/1991.iwpt.xml:126]: minor typo

File: data/xml/1993.iwpt.xml [hunk 7]
```diff
@@ -0,0 +1,236 @@
...
+      <author><first>Rieks</first><last>op den Akker</last></author>
+      <pages>267-276</pages>
+      <url>1993.iwpt-1.21</url>
+      <abstract>Head-Corner (HC) parsing has come up in computational linguistics a few years ago, motivated by linguistic arguments. This idea is a heuristic, rather than a fail-safe principle, hence it is relevant indeed to consider the worst-case behaviour of the HC parser. We define a novel predictive head-corner chart parser of cubic time complexity. We start with a left-corner (LC) chart parser, which is easier to understand. Subsequently, the LC chart parser is generalized to an HC chart parser. It is briefly sketched how the parser can be enhanced with feature structures.</abstract>
+    </paper>
+    <paper id="22">
+      <title>Parsing <fixed-case>E</fixed-case>nglish with a Link Grammar</title>
+      <author><first>Daniel D.</first><last>Sleator</last></author>
+      <author><first>Davy</first><last>Temperley</last></author>
+      <pages>277-292</pages>
```
Comment (mjpost) [inline data/xml/1993.iwpt.xml:191]: We should add a name variant here for David Temperley

File: data/xml/1995.iwpt.xml [hunk 8]
```diff
@@ -0,0 +1,287 @@
...
+    <paper id="4">
+      <title>Parallel Parsing: Different Distribution Schemata for Charts</title>
+      <author><first>Jan W.</first><last>Amtrup</last></author>
+      <pages>12-13</pages>
+      <url>1995.iwpt-1.4</url>
+      <abstract></abstract>
+    </paper>
+    <paper id="5">
+      <title>A Fuzzy APPROACH TO ERRONEOUS INPUTS IN CONTEXT-FREE LANGUAGE RECOGNITION</title>
+      <author><first>Peter R.J.</first><last>Asveld</last></author>
```
Comment (mjpost) [inline data/xml/1995.iwpt.xml:60]: caps

File: data/xml/1995.iwpt.xml [hunk 9]
```diff
@@ -0,0 +1,287 @@
...
+      <author><first>Mark</first><last>Moll</last></author>
+      <author><first>Anton</first><last>Nijholt</last></author>
+      <pages>10-11</pages>
+      <url>1995.iwpt-1.3</url>
+      <abstract>The analysis of natural language in the context of keyboard-driven dialogue systems is the central issue addressed in this paper. A module that corrects typing errors, performs domain-specific morphological analysis is developed. A parser for typed unification grammars is designed and implemented in C++; for description of the lexicon and the grammer a specialised specification language is developed. It is argued that typed unification grammars and especially the newly developed specification language are convenient formalisms for describing natural language use in dialogue systems. Research on these issues is carried out in the context of the SCHISMA project, a research project in linguistic engineering; participants in SCHISMA are KPN Research and the University of Twente.</abstract>
+    </paper>
+    <paper id="4">
+      <title>Parallel Parsing: Different Distribution Schemata for Charts</title>
+      <author><first>Jan W.</first><last>Amtrup</last></author>
```
Comment (mjpost) [inline data/xml/1995.iwpt.xml:53]: probably should make this block caps for consistency with the rest of the volume

File: data/xml/1995.iwpt.xml [hunk 10]
```diff
@@ -0,0 +1,287 @@
...
+    <paper id="23">
+      <title>A FORMALISM AND A PARSER FOR LEXICALISED DEPENDENCY GRAMMARS</title>
+      <author><first>Alexis</first><last>NASR</last></author>
+      <pages>186-195</pages>
+      <url>1995.iwpt-1.23</url>
+      <abstract></abstract>
+    </paper>
+    <paper id="24">
+      <title>Error-tolerant Finite State Recognition</title>
+      <author><first>Kemal</first><last>Oflazer</last></author>
```
Comment (mjpost) [inline data/xml/1995.iwpt.xml:210]: caps

File: data/xml/1995.iwpt.xml [hunk 11]
```diff
@@ -0,0 +1,287 @@
...
+      <abstract>A reductionistic statistical framework for part-of-speech tagging and surface syntactic parsing is presented that has the same expressive power as the highly successful Constraint Grammar approach, see [Karlsson et al. 1995]. The structure of the Constraint Grammar rules allows them to be viewed as conditional probabilities that can be used to update the lexical tag probabilities, after which low-probability tags are repeatedly removed. Experiments using strictly conventional information sources on the Susanne and Teleman corpora indicate that the system performs as well as a traditional HMM-based part-of-speech tagger, yielding state-of-the-art results. The scheme also enables using the same information sources as the Constraint Grammar approach, and the hope is that it can improve on the performance of both statistical taggers and surface-syntactic analyzers.</abstract>
+    </paper>
+    <paper id="26">
+      <title>A Corpus-based Probabilistic Grammar with Only Two Non-terminals</title>
+      <author><first>Satoshi</first><last>SEKINE</last></author>
+      <author><first>Ralph</first><last>GRISHMAN</last></author>
+      <pages>216-223</pages>
```
Comment (mjpost) [inline data/xml/1995.iwpt.xml:226]: caps on last names...

File: data/xml/1995.iwpt.xml [hunk 12]
```diff
@@ -0,0 +1,287 @@
...
+    </paper>
+    <paper id="27">
+      <title>HEURISTICS AND PARSE RANKING</title>
+      <author><first>B.</first><last>Srinivas</last></author>
+      <author><first>Christine</first><last>Doran</last></author>
```
Comment (mjpost) [inline data/xml/1995.iwpt.xml:233]: Is this [Srinivas Bangalore](https://www.aclweb.org/anthology/people/s/srinivas-bangalore/)? It seems the name is backwards, though I guess it's in the variants file.

File: data/xml/1995.iwpt.xml [hunk 13]
```diff
@@ -0,0 +1,287 @@
...
+      <author><first>Seth</first><last>Kulick</last></author>
+      <pages>224-233</pages>
+      <url>1995.iwpt-1.27</url>
+      <abstract>There are currently two philosophies for building grammars and parsers – Statistically induced grammars and Wide-coverage grammars. One way to combine the strengths of both approaches is to have a wide-coverage grammar with a heuristic component which is domain independent but whose contribution is tuned to particular domains. In this paper, we discuss a three-stage approach to disambiguation in the context of a lexicalized grammar, using a variety of domain independent heuristic techniques. We present a training algorithm which uses hand-bracketed treebank parses to set the weights of these heuristics. We compare the performance of our grammar against the performance of the IBM statistical grammar, using both untrained and trained weights for the heuristics.</abstract>
+    </paper>
+    <paper id="28">
+      <title>STOCHASTIC PARSE-TREE RECOGNITION BY A PUSHDOWN AUTOMATON</title>
+      <author><first>Frédéric</first><last>TENDEAU</last></author>
+      <pages>234-249</pages>
```
Comment (mjpost) [inline data/xml/1995.iwpt.xml:242]: caps on last name

**Comment by mjpost** [comment]
I just noticed you had provided the tarball link. They are in place, e.g., https://aclweb.org/anthology/1995.iwpt-1.1.pdf.

Do we want to add the full volume PDFs, too?

File: data/xml/1995.iwpt.xml (hunk 12 above)
Comment (kilian-gebhardt) [reply data/xml/1995.iwpt.xml:233]: it is

**Comment by kilian-gebhardt** [comment]
Thanks @danielgildea for taking over.

@mjpost I fixed all the mistakes except for the the title BLOCK CAPS in the 1995 volume. Do we need this to be consistent in the xml-Files or can capitalization also be handled by scripts that create the website and bibtex-files?

Concerning the full volume PDFs – we can share those of course but they are comparably large (60-106 MB).

**Comment by mjpost** [comment]
Hi @kilian-gebhardt—I think we should add them; they're useful to some people and the size isn't a big issue.

If you can name them as "{year}.iwpt-1.pdf" and share them with me, I'll add them in.

**Comment by kilian-gebhardt** [comment]
@mjpost Here you go: https://wwwtcs.inf.tu-dresden.de/~kilian/assets/IWPT/iwpt.tar.gz

**Comment by kilian-gebhardt** [comment]
One remark: the 1995.iwpt-1 is currently not listed under https://www.aclweb.org/anthology/events/ws-1995/

**Pull request merged**